03 Writing Algorithms/34 Algorithm Framework/08 Insight Manager/02 Add Insights.html

```diff
@@ -2,12 +2,16 @@
 
 <p>To add an insight to an <code>InsightCollection</code>, call the <code class="csharp">Add</code><code class="python">add</code> method.</p>
 <div class="section-example-container">
-    <pre class="csharp">_insightCollection.Add(insight);</pre>
-    <pre class="python">self.insight_collection.add(insight)</pre>
+    <pre class="csharp">// Add an Insight to the InsightCollection, which manages all insights in the algorithm for seamless integration and accessibility of forecast data anywhere in the algorithm.
+_insightCollection.Add(insight);</pre>
+    <pre class="python"># Add an Insight to the InsightCollection, which manages all insights in the algorithm for seamless integration and accessibility of forecast data anywhere in the algorithm.
+self.insight_collection.add(insight)</pre>
 </div>
 
 <p>To add a list of insights, call the <code class="csharp">AddRange</code><code class="python">add_range</code> method.</p>
 <div class="section-example-container">
-    <pre class="csharp">_insightCollection.AddRange(insights);</pre>
-    <pre class="python">self.insight_collection.add_range(insights)</pre>
+    <pre class="csharp">// Add a list of Insight objects to the InsightCollection, which manages all insights in the algorithm for comprehensive data integration and accessibility.
```

Comment:
Please don't use chatGPT to write comments. "for comprehensive data integration and accessibility".